2024-12-23 02:04:10,042 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 120, in startMenu
    raise ValueError("You can only choose 1 or 2")
ValueError: You can only choose 1 or 2
2024-12-23 02:04:17,450 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 120, in startMenu
    raise ValueError("You can only choose 1 or 2")
ValueError: You can only choose 1 or 2

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 985, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 35, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 124, in startMenu
    self.startMenu()  # Tekrar baþlangýç menüsünü çaðýr
    ^^^^^^^^^^^^^^^^
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 111, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 12:54:08,203 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 984, in <module>
    main = Main()
           ^^^^^^
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 45, in __init__
    self._faculties = self._data_management.getAllFaculties()
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\DataManagement.py", line 179, in getAllFaculties
    for data in datas:
TypeError: 'NoneType' object is not iterable
2024-12-23 12:54:47,678 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 120, in startMenu
    raise ValueError("You can only choose 1 or 2")
ValueError: You can only choose 1 or 2
2024-12-23 12:55:18,067 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 120, in startMenu
    raise ValueError("You can only choose 1 or 2")
ValueError: You can only choose 1 or 2

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 985, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 35, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 124, in startMenu
    self.startMenu()  # Tekrar baþlangýç menüsünü çaðýr
    ^^^^^^^^^^^^^^^^
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 111, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:01:33,743 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 985, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 35, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 114, in startMenu
    self.login()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 891, in login
    self.user_type = input()
                     ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:04:21,003 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 985, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 35, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 111, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:05:38,406 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 985, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 35, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 111, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:06:33,894 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 986, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 36, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 112, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:08:57,667 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 986, in <module>
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 36, in main
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 112, in startMenu
    try:
         
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:09:10,057 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 985, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 35, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 111, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:11:38,665 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 985, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 35, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 111, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:12:21,814 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 985, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 35, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 111, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:14:59,215 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 985, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 35, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 111, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:15:07,803 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 988, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 38, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 114, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:15:17,894 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 988, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 38, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 114, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:15:39,472 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 988, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 38, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 114, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:15:57,365 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 988, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 38, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 114, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:17:49,616 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 988, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 38, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 114, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:23:42,353 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 996, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 38, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 114, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:27:55,535 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 996, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 38, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 114, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:29:38,163 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 996, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 38, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 114, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:33:56,201 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 996, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 38, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 114, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:43:05,501 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 996, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 38, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 114, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
2024-12-23 13:45:46,642 - exception - ERROR - An error occurred!
Traceback (most recent call last):
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 996, in <module>
    main.main()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 38, in main
    self.startMenu()
  File "C:\Users\omery\PycharmProjects\CSE3063F24P1_GRP11\pythonCode\Main.py", line 114, in startMenu
    choice = input()
             ^^^^^^^
  File "<frozen codecs>", line 319, in decode
KeyboardInterrupt
